src/staking-v3/components/dapp/DappV3.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <div v-if="dapp && dapp.extended" class="container--dapp-staking">
+    <back-to-page :text="$t('dappStaking.stakePage.backToDappList')" :link="Path.DappStaking" />
+    <dapp-avatar :dapp="dapp" />
+    <dapp-statistics :dapp="dapp" />
+    <dapp-images :dapp="dapp" />
+    <builders :dapp="dapp" />
+
+    <div class="text--title">{{ $t('stakingV3.dapp.overview') }}</div>
+    <div class="row--project-overview">
+      <project-overview :dapp="dapp" />
+      <project-details :dapp="dapp" class="project--details" />
+    </div>
+    <!-- <dapp-stats-charts :dapp="dapp" /> -->
+    <div class="bottom--links">
+      <div class="bottom--links__inner">
+        <button
+          class="button--start-staking"
+          :disabled="isZkEvm"
+          @click="navigateToVote(dapp.chain.address)"
+        >
+          {{ $t('dappStaking.dappPage.stakeOrSwitchTo') }} {{ dapp.extended.name }}
+        </button>
+
+        <div class="column--action">
+          <!-- TODO: add logic -->
```

Comment:
When and how do we implement the logic here?

Reply:
Let's leave this for the future PR